README_dev.md

```diff
@@ -119,7 +119,9 @@ npm run test:local -- testSuiteName
 2. change into the directory `cd connector`
 3. install the npm dependencies `npm i`
 4. build the connector `npm run build`
-5. create a config file (for example `local.config.json`)
+5. install the connector as cli locally `npm link .`
```

Comment:
If this is an optional step, it should be marked as such.

Reply:
I would go further and NOT recommend to do that.